lib/ansible/modules/cloud/digital_ocean/digital_ocean_tag.py

```diff
@@ -129,10 +130,10 @@ def __init__(self, resp, info):
     def json(self):
         if not self.body:
             if "body" in self.info:
-                return json.loads(self.info["body"])
+                return json.loads(to_text(self.info["body"]))
```

Comment:
The call to to_text here and on line 136 should set the error handler to surrogate_or strict like this:
``` python
return json.loads(to_text(self.info["body"], errors='surrogate_or_strict'))
```

The default error handler is replace which would turn non-decodable values into a replacement character (For instance, "?").  Because we're dealing with what should be well formed data, we'd rather throw an exception here if we get undecodable data than to go ahead with somethng that's potentially corrupted.